### ISSUE TITLE
Val Cannot be Reassigned

### ISSUE THREAD

**Issue by abueide** [beryx/badass-runtime-plugin#84]
Hey so I get val cannot be reassigned when trying to set the jvm args. jvmArgs.set(listOf("")) results in an unsupported operation exception.

![image](https://user-images.githubusercontent.com/19354425/117586925-dfde9000-b0e0-11eb-83fe-b168ce08d18f.png)

**Comment by siordache** [comment]
Use:
`jvmArgs.addAll(jvmOptions)`

I fixed the example in the documentation.

**Comment by abueide** [comment]
@siordache Wait sorry that also just gives me an unsupported operation exception

```
* Where:
Build file 'C:\Users\Andrew\programming\projects\harry-plotter\build.gradle.kts' line: 80

* What went wrong:
java.lang.UnsupportedOperationException (no error message)
```

**Comment by siordache** [comment]
That was really a bug. It's fixed in 1.12.5.
Both `jvmArgs = jvmOptions` and `jvmArgs.addAll(jvmOptions)` are now accepted.
However, in harry-plotter you don't need to set jvmArgs, because the plugin uses by default the value of `applicationDefaultJvmArgs`.

**Comment by abueide** [comment]
interesting I didn't know that, thank you!
